generators/chipyard/src/main/scala/config/NoCoreConfigs.scala

```diff
@@ -17,3 +17,9 @@ class NoCoresConfig extends Config(
   new chipyard.config.WithNoDebug ++
   new chipyard.config.WithNoPLIC ++
   new chipyard.config.AbstractConfig)
+
+// A config that uses a empty chiptop module with no rocket-chip soc components
+class EmptyChipTopConfig extends Config(
+  new chipyard.example.WithEmptyChipTop ++
+  new chipyard.config.AbstractConfig             // since we aren't using rocket-chip, this doesn't do anything
```

Comment:
Then why keep this then?

Reply:
Its kind of ugly... you can't do 

```
extends Config(
  new chipyard.example.WithEmptyChipTop
)
```

By itself. You'd have to do 

```
extends chipyard.example.withEmptyChipTop
````


But this is confusing to users. I elected to follow the pattern